.github/workflows/publish-nuget.yml

```diff
@@ -39,7 +38,7 @@ on:
       timeout-minutes:
         type: number
         default: 15
-        description: Configuration for the timeout-minutes parameter of the workflow. The 360 is GitHub's default.
+        description: Configuration for the timeout-minutes parameter of the workflow. 360 is GitHub's default.
```

Comment:
This is about the default timeout for every GitHub workflow, which is 360. This is added as information for context.

Reply:
It's still misleadingly phrased. In that case you'd want to write
```suggestion
        description: Configuration for the timeout-minutes parameter of the workflow. GitHub's default timeout is 360.
```
but I don't think it merits mentioning because it's irrelevant for this specific input (whose default value is 15 and not 360 like with the other `timeout-minutes`s above).

Reply:
Yeah, fair enough.